Formula/swiftgen.rb

```diff
@@ -16,6 +16,13 @@ class Swiftgen < Formula
   depends_on xcode: ["13.0", :build]
   depends_on :macos
 
+  resource("testdata") do
+    url "https://github.com/SwiftGen/SwiftGen.git",
+      tag:      "6.5.1",
+      revision: "d498d285867f42ef0c74c5ebd9ddced747831372",
+      branch:   "develop"
+  end
```

Comment:
I understand the desire to avoid having to package the test fixtures in the bottle just for the test, but it also seems excessive to clone the repo again just for the test.

Maybe it would be simpler (not necessarily for now) if you could add the test fixtures as a separate asset along with the release, so that these can be downloaded independently?

Alternatively, if your test fixtures are available in the release tarball, we could use that here instead. (Actually if we could switch the stable `url` to a tarball that would be useful also.)

Reply:
I'd love to do that, GitHub provides tarballs for any branch and tag on a repository, for example:
> "https://github.com/SwiftGen/SwiftGen/archive/6.5.1.tar.gz"

But then Homebrew complains about missing checksums. How are we supposed to provide a checksum for a file we do not own (and is generated on the fly)?